### PR TITLE
Creating Title-Subtitle style Land Entry message

### PR DESCRIPTION
**Pull request by Dweblenod**
Does not replace existing chat message, just another version! Introduces Carima font used by Homestuck for their Land Entry messages

### CODE REVIEW

File: src/main/resources/assets/minestuck/font/carima.ttf
Comment (kirderf1) [inline src/main/resources/assets/minestuck/font/carima.ttf:1]: What is this font licensed under? Is there somewhere we can note which license that applies?

File: src/main/java/com/mraof/minestuck/data/MinestuckEnUsLanguageProvider.java [hunk 1]
```diff
@@ -1699,7 +1699,8 @@ protected void addTranslations()
 		addLand(MonstersLandType.MONSTERS, "Monsters");
 		addLand(TowersLandType.TOWERS, "Towers");
 		add(LandTypePair.FORMAT, "Land of %s and %s");
-		add(SburbHandler.LAND_ENTRY, "Welcome to %s.");
+		add(SburbHandler.CHAT_LAND_ENTRY, "Welcome to %s.");
+		add(SburbHandler.TITLE_LAND_ENTRY, "%s");
```
Comment (kirderf1) [inline src/main/java/com/mraof/minestuck/data/MinestuckEnUsLanguageProvider.java:1703]: What is the point to this specific lang entry?

Comment (Dweblenod) [reply src/main/java/com/mraof/minestuck/data/MinestuckEnUsLanguageProvider.java:1703]: I was under the impression we wanted to keep it translatable, in which case two arguments are needed unless I can just "%s" into SburbHandler

File: src/main/resources/assets/minestuck/font/carima.ttf
Comment (Dweblenod) [reply src/main/resources/assets/minestuck/font/carima.ttf:1]: It has been determined that we would be distributing this, which goes against copyright. Removing

File: src/main/java/com/mraof/minestuck/data/MinestuckEnUsLanguageProvider.java (hunk 1 above)
Comment (kirderf1) [reply src/main/java/com/mraof/minestuck/data/MinestuckEnUsLanguageProvider.java:1703]: Keeping it translatable? Two arguments are needed? I'm not sure what problem you're trying to solve.

Comment (Dweblenod) [reply src/main/java/com/mraof/minestuck/data/MinestuckEnUsLanguageProvider.java]: Issue resolved, thanks for help!

**Review by kirderf1 — APPROVED** [review]

**Pull request merged**